frontend/src/lib/k8s/index.test.ts

```diff
@@ -55,3 +55,148 @@ describe('Class tests', () => {
     });
   });
 });
+
+const labelSelector: LabelSelector = {
+  matchLabels: {
+    app: 'test',
+    label1: 'value1',
+  },
+};
+
+describe('Label selector', () => {
+  test('Match labels alone', () => {
+    const query = labelSelectorToQuery(labelSelector);
+    expect(query).toBe('app=test,label1=value1');
+  });
+
+  test('Match labels and match expressions', () => {
+    const query = labelSelectorToQuery({
+      ...labelSelector,
+      matchExpressions: [
+        {
+          key: 'label2',
+          operator: 'In',
+          values: ['value2', 'value3'],
+        },
+      ],
+    });
+    expect(query).toBe('app=test,label1=value1,label2 in (value2,value3)');
+  });
+
+  test('Match expression In', () => {
+    const query = labelSelectorToQuery({
+      matchExpressions: [
+        {
+          key: 'label2',
+          operator: 'In',
+          values: ['value2', 'value3'],
+        },
+      ],
+    });
+    expect(query).toBe('label2 in (value2,value3)');
```

Comment:
Is there supposed to be a space?

- `'label2 in (value2,value3)'`
- VS `'label2 in (value2, value3)'`

I ask because the examples I see have spaces. https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#list-and-watch-filtering

Reply:
Both work with kubectl. Let's keep the one without a space since this will save a few bytes when sending the request over.